fake_switches/switch_configuration.py

```diff
@@ -217,6 +217,7 @@ def __init__(self, vlan_id, *args, **kwargs):
         self.ip_proxy_arp = True
         self.unicast_reverse_path_forwarding = False
         self.load_interval = None
+        self.mpls_ip = None
```

Comment:
Should be True or False

Reply:
Dépendant de ce qu'on veux faire, souvent les switch on 3 etats:
None: unset / default
True: explicitly enabled
False: explicitly disabled

Reply:
Après discussion avec simon, mettons True (enabled par défaut)